src/spinner/internal.tsx

```diff
@@ -1,28 +1,55 @@
 // Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
 // SPDX-License-Identifier: Apache-2.0
 import clsx from 'clsx';
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import { getBaseProps } from '../internal/base-component';
 import styles from './styles.css.js';
 import { SpinnerProps } from './interfaces';
 import { InternalBaseComponentProps } from '../internal/hooks/use-base-component';
+import LiveRegion from '../internal/components/live-region';
 
 interface InternalSpinnerProps extends SpinnerProps, InternalBaseComponentProps {}
 
 export default function InternalSpinner({
   size = 'normal',
   variant = 'normal',
+  loadingAriaLiveText,
   __internalRootRef,
   ...props
 }: InternalSpinnerProps) {
   const baseProps = getBaseProps(props);
 
+  const [liveRegionText, setliveRegionText] = useState('');
+  /* Results in the message repeating at twice the value of messageToggleTiming.
+   * In this case 4 seconds was the interval that "felt right" to the a11y team
+   */
+  const messageToggleTiming = 2000;
```

Comment:
Is it 2 seconds or 4 seconds?

Reply:
It is 4 seconds. It changes state every 2 seconds but only only one state announces, the other clears it. I'll clarify this,

Reply:
See: https://github.com/cloudscape-design/components/pull/316#discussion_r984650242